docs/lua-meta/globals.lua

```diff
@@ -54,6 +54,35 @@ c2.ChannelType = {
 
 -- End src/common/Channel.hpp
 
+-- Begin src/controllers/plugins/api/Accounts.hpp
+
+
+c2.accounts = {}
+c2.accounts.twitch = {}
+
+---@class c2.accounts.twitch.TwitchAccount
+c2.accounts.twitch.TwitchAccount = {}
+
+---@return string user_name The (login) name of the account
+function c2.accounts.twitch.TwitchAccount:user_name() end
```

Comment:
Just noting, not sure if I want any changes:
Should we ensure plugins are consistent with Twitch's terminology?
Login meaning the lowercased version of the username
Display Name meaning the potentially upercased version, potentially with CJK characters.

so, should this be called `user_login`?